repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,6 +4,6 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "cd hdfs-mesos* && ./bin/hdfs-mesos",
+  "cmd": "{{#hdfs/custom-config}}echo {{hdfs/custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs/custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
```

Comment:
That would need to be documented in the official installation guide(s) and other tutorials.